src/e3/diff.py

```diff
@@ -37,6 +37,7 @@ def diff(
     item1name: str = "expected",
     item2name: str = "output",
     ignore_white_chars: bool = True,
+    context: int = 1,
```

Comment:
Apart from @grouigrokon 's comment, my only point concerns the type of context. It's supposed to be an unsigned int (this type doesn't exist in python).

I haven't yet found a way of specifying this type that works with mypy and doesn't require more code (the `Annotated` type does exist, but I've tested it and our mypy didn't detect an error).

Also, so that it's clear to everyone, can you at least add in the comment that it's a positive int?

Maybe also add an `assert context >= 0` or an `if context <= 0: raise ValueError` (I don't really know if this will help mypy understand what's going on, but at least it's explicit 😄)

Reply:
Done.